fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/ContainerAuthToken.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.auth.common;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.apache.shiro.authc.AuthenticationToken;
+import org.fcrepo.auth.common.ContainerRolesPrincipalProvider.ContainerRolesPrincipal;
+import org.slf4j.Logger;
+
+/**
+ * @author peichman
+ */
+public class ContainerAuthToken implements AuthenticationToken {
+
+    private static final Logger log = getLogger(ContainerAuthToken.class);
+
+    public static final String AUTHORIZED = "AUTHORIZED";
+
+    private String servletUsername;
+
+    private Set<ContainerRolesPrincipal> servletRoles;
+
+    /**
+     * @param servletUsername
```

Comment:
[WARNING] Javadoc Warnings
[WARNING] /home/awoods/programming/java/apps/fedora/futures/fcrepo4/fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/ContainerAuthToken.java:43: warning: no description for @param
[WARNING] * @param servletUsername